### ISSUE TITLE
A question of the pre-trained model

### ISSUE THREAD

**Issue by aiwen7** [ChristopherLu/milliEgo#5]
Hello,
Thank you for providing the code for the interesting project.
I have a problem with the performance distinction when training the model w and w/o pre-trained models.
In practice, we trained the official code under default configuration with the pre-trained CNN model and from scratch (without pre-trained models), respectively. In our case, we took out nine training sequences as the validation set since validation sequences are not found in the dataset. We found that the model with pre-trained CNN has a better performance than the model from scratch in terms of visualization and quantification results. But both are generally worse than the provided model, i.e., '140'. Therefore, we want to know how the pre-trained models (i.e., 'cnn.h5' and '140') are trained and generated. 
I will be grateful for any help you can provide!
